src/app/tabs/report/report.page.html

```diff
@@ -1,9 +1,74 @@
 <ion-header>
   <ion-toolbar>
     <ion-title>Report</ion-title>
+    <ion-button (click)="onClickLogout()" slot="end" fill="clear" color="tertiary">
+      <ion-icon slot="icon-only" name="log-out-outline"></ion-icon>
+    </ion-button>
   </ion-toolbar>
```

Comment:
Please create header section separate and start using across the application , instead of declaring same header section in every tab page.

Reply:
Use custom element concept.